support-payment-api/src/main/scala/backend/AmazonPayBackend.scala

```diff
@@ -67,6 +67,12 @@ class AmazonPayBackend(
       )
     }
 
+    def isValidEmail(email: String): Boolean = {
+      // Use a regular expression to check if the email address is valid and does not contain a comma
+      val emailRegex = "^[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\\.[A-Z|a-z]{2,}$"
```

Comment:
There was a decision recently not to use a general regex for email validation, and instead to specifically guard against commas: https://github.com/guardian/support-frontend/pull/5617